internal/rbd/rbd_journal.go

```diff
@@ -568,7 +585,13 @@ func RegenerateJournal(
 	if rbdVol.JournalPool == "" {
 		rbdVol.JournalPool = rbdVol.Pool
 	}
-	volJournal = journal.NewCSIVolumeJournal("default")
+	// FIXME: this works only on Kubernetes, how do other CO supply metadata?
```

Comment:
I think we can drop above comment. The reason being in many areas of code, we can only work CO kubernetes.